react-toggle-switch/my-app/src/App.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import './toggleSwitch.css';
+import './App.css';
+import { ToggleSwitch } from './toggleSwitch';
+
+export default function App() {
+  let className = 'toggle-switch';
+  let stateLabel = 'OFF';
+  const [isOn, setIsOn] = useState(false);
+
+  const handleClick = () => {
+    setIsOn(!isOn);
+  };
+
+  if (isOn === true) {
```

Comment:
Usually you wouldn't check a boolean variable explicitly against `true` or `false`. Instead you would rely on the truthiness feature of JavaScript and just say `if (isOn)` or `if (!isOn)`.